xml/admin_nfsganesha.xml

```diff
@@ -389,6 +405,80 @@ EXPORT {
     </variablelist>
    </sect3>
   </sect2>
+
+  <sect2 xml:id="ganesha-getting-exports">
+   <title>Getting Exports Configuration</title>
```

Comment:
I get what this means, but it's not a great title. How about just "Exports Configuration" or "Exporting Configurations"

Reply:
'Exports Configuration' was the previous section and it dealt with how to CREATE a configuration for a NFS export.
This one is bout how to GET the configuration for already existing exports as it is not transparent in a config file but rather  RADOS object.

Reply:
Then I think "get" is too vague, as it didn't convey your meaning to me at all. @lproven is usually good at the wordy words here, but at a stab, how about: "Retrieving the Exports Configuration" or "Recover Exports Configuration"